charts/cluster-api-runtime-extensions-nutanix/templates/mindthegap.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024 Nutanix. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+# This file contains the manifests to run a mindthegap deployment which contains helm charts for our addons.
+# The pod is built via goreleaser with configuration from hack/addons.
+#
+{{ if .Values.selfHostedRegistry }}
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: mindthegap-selfsigned
+  namespace: {{ .Release.Namespace }}
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: mindthegap-tls
+  namespace: {{ .Release.Namespace }}
+spec:
+  dnsNames:
+  - mindthegap.{{ .Release.Namespace }}.svc
+  - mindthegap.{{ .Release.Namespace }}.svc.cluster.local
+  issuerRef:
+    name: mindthegap-selfsigned
+    kind: Issuer
+  secretName: mindthegap-tls
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mindthegap
```

Comment:
It would be good to use generic name like `chart-repository` instead of `mindthdgap` in deployments, cert resources etc, so that we can change the backend from mindthegap to something else if needed.

Reply:
yep- working on changing to just `chart-repository`